lib/internal/Magento/Framework/Url/RouteParamsResolver.php

```diff
@@ -112,7 +112,7 @@ public function setRouteParams(array $data, $unsetOldParams = true)
             } else {
                 $this->setRouteParam(
                     $this->getEscaper()->encodeUrlParam($key),
-                    $this->getEscaper()->encodeUrlParam($value)
+                    $this->getEscaper()->encodeUrlParam((string)$value)
```

Comment:
@ihor-sviziev With new changes we can rollback changes for this file

Reply:
No need, it still suppose to be a string, right?